CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # Change Log
 
+# [1.6.0 (November 26th, 2021)](https://github.com/eclipse/eclipse.jdt.ls/milestone/96?closed=1)
+ * enhancement - Stop generating metadata files at project's root. See [#1900](https://github.com/eclipse/eclipse.jdt.ls/pull/1900).
+ * enhancement - Quickfixes should be available at the line level. See [#1908](https://github.com/eclipse/eclipse.jdt.ls/pull/1908).
+ * enhancement - Add `Generate Constructors` to Show Fixes for type declaration. See [#1937](https://github.com/eclipse/eclipse.jdt.ls/pull/1937).
+ * enhancement - Add `Override/Implement methods` to Show Fixes for type declaration. See [#1932](https://github.com/eclipse/eclipse.jdt.ls/pull/1932).
```

Comment:
Add a new one "Always show Organize imports in Quick Fixes for import declaration #1936"